### PR TITLE
Edit CSS animations

### LINKED ISSUE #438: [Edit] CSS : Animations
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

Make changes in this file: **content/css/concepts/animations/animations.md**

Let's change the `@keyframes` rule to better demonstrate how CSS animations allow for multiple style changes to be implemented at different points in the frame-to-frame transition. Something like:

```css
@keyframes color-change {
  from {
    color: purple;
  }
  25% {
    color: yellow;
  }
  50% {
    color: red;
  }
  75% {
    color: blue;
  }
  to {
    color: yellow:
  }
}
```

And then, in the `h1` selector, let's change the `color` value to `purple`.

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Assignee(s) added

### PR DESCRIPTION
**Pull request by yangc95**
### Description

Closes #438 for requested edits to [CSS animations](https://www.codecademy.com/resources/docs/css/animations)

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/animations/animations.md [hunk 1]
```diff
@@ -20,22 +20,31 @@ To use CSS animation, some keyframes for the animation must be specified.
...
 ```css
 /* The animation */
 @keyframes color-change {
   from {
+    color: purple;
+  }
+  25% {
     color: yellow;
   }
+  50% {
```
Comment (SSwiniarski) [inline content/css/concepts/animations/animations.md:34]: With this edit, "yellow" appears twice in the animation. Is that intentional?

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@yangc95, I reviewed this and only had one comment.

File: content/css/concepts/animations/animations.md (hunk 1 above)
Comment (KTom101) [reply content/css/concepts/animations/animations.md:34]: @SSwiniarski I think that's the edit that was requested in the [linked issue](https://github.com/Codecademy/docs/issues/438).

Comment (SSwiniarski) [reply content/css/concepts/animations/animations.md:34]: @KTom101 That's how the suggested edit was written, but I think the actual issue was just having an animation with multiple steps. I just think having yellow twice is odd.

Comment (KTom101) [reply content/css/concepts/animations/animations.md:34]: @SSwiniarski I see what you mean.

Comment (yangc95) [reply content/css/concepts/animations/animations.md:34]: @SSwiniarski @KTom101 How about changing yellow -> orange on line 32?

Comment (SSwiniarski) [reply content/css/concepts/animations/animations.md:34]: @yangc95, That would be good.

File: content/css/concepts/animations/animations.md [hunk 2]
```diff
@@ -20,22 +20,31 @@ To use CSS animation, some keyframes for the animation must be specified.
...
+To get an animation to work, the animation must be bound to an element and the `animation-duration` should be defined. In this example code, the `<h1>` element will load as purple and gradually change to yellow over the course of 5 seconds.
 
 ```css
 /* The animation */
 @keyframes color-change {
   from {
+    color: purple;
+  }
+  25% {
     color: yellow;
```
Comment (yangc95) [inline content/css/concepts/animations/animations.md:32]:
```suggestion
    color: orange;
```

**Review by SSwiniarski — APPROVED** [review]
LGTM!

**Pull request merged**